src/components/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from 'react';
+import { Status } from '../services/EnumStatusFilter';
+import { TodoContext } from './TodoContext';
+
+type Props = {};
+
+export const Footer: React.FC<Props> = () => {
+  const {
+    deleteAllCompleted,
+    count,
+    selectTodoFilteredList,
+    setSelectTodoFilteredList,
+    todos,
+  } = useContext(TodoContext);
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${count} items left`}
+      </span>
+
+      <ul
+        className="filters"
+        data-cy="todosFilter"
+      >
+        <li>
+          <a
```

Comment:
why it is a link?

Reply:
![1212](https://github.com/mate-academy/react_todo-app/assets/97402422/4ce969f6-71ed-4846-b7f1-a3b5cf0a16c4)